bitsandbytes/functional.py

```diff
@@ -439,6 +439,11 @@ def is_on_gpu(tensors):
     return on_gpu
 
 
+def get_tensor_stream(tensor: Tensor) -> int:
+    stream = torch.cuda.current_stream(tensor.device).cuda_stream
+    return stream
+
+
```

Comment:
It would be cleaner here to use `ct.c_void_p` instead of uint64. In fact `._as_parameter_` is defined on PyTorch's `Stream` wrapper, so it should just work with ctypes function calls.

E.g. change this to `return torch.cuda.current_stream(tensor.device)` and replace `ct.c_uint64(stream)` in the C invocations with just `stream`. This would effectively be the same as `ct.c_void_p(stream.cuda_stream)` and could remove the need for the `reinterpret_cast` etc.

Reply:
thank you very much, I will handling it asap